modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upload/DDMFormUploadValidator.java

```diff
@@ -41,6 +52,28 @@ public long getGuestUploadMaximumFileSize() {
 			_FILE_LENGTH_MB;
 	}
 
+	public void validateFileExtension(String fileName)
+		throws FileExtensionException {
+
+		boolean validFileExtension = false;
+
+		for (String guestUploadFileExtension : getGuestUploadFileExtensions()) {
+			if (StringUtil.equalsIgnoreCase(
+					FileUtil.getExtension(fileName),
+					StringUtil.trim(guestUploadFileExtension))) {
+
+				validFileExtension = true;
+
+				break;
+			}
+		}
```

Comment:
@carolmariaabb, instead of use a `break` here, how about try to use something like [this](https://mkyong.com/java8/java-8-stream-findfirst-and-findany/)?